### ISSUE TITLE
"npm install" security warning not reported by dependency check

### ISSUE THREAD

**Issue by carloreggiani** [dependency-check/DependencyCheck#856]
Using `-enableExperimental` with version 2.1.0, it is non clear to me how to use information from depcheck report.

I'm doing some verification comparing `npm install` security warning messages and reports generated by devcheck: there are security warning on component not identified in the dependency check report.

Here one example:

From `npm install`:

`npm WARN deprecated jsonwebtoken@0.3.0: Critical vulnerability fix in v5.0.0. See https://auth0.com/blog/2015/03/31/critical-vulnerabilities-in-json-web-token-libraries/`

In the depcheck repo:

```
jsonwebtoken/package.json
Evidence: 
package.json | author.name | auth0
package.json | description | JSON Web Token implementation (symmetric and asymmetric)
package.json | name | jsonwebtoken
package.json | name_project | jsonwebtoken_project
package.json | version | 0.3.0

Identifiers:
None
```

Thanks for any info!

Carlo

**Comment by jeremylong** [comment]
Dependency-check 2.x has 2 analyzers for Node. One is experimental and will likely be removed soon as it just isn't performing as hoped. The other is just a wrapper for the Node Security Project (NSP). The NSP analyzer will likely report the same warnings as reported by `npm install`. Try version 2.1.1 as a few bugs were found that have been fixed.

**Comment by carloreggiani** [comment]
Executed analisys on the same project with 2.1.1: unfortunally no change in my results.

I noted no result at all (also with 2.1.1) without `--enableExperimental`: it means any kind of problem in the wrapper for NSP? I'm behind a proxy, but with command line parameters it's working fine to access NVD files.

I'm not a developer: hope to help as a tester :)

**Comment by stevespringett** [comment]
@carloreggiani I'm don't know what checks are actually performed using `npm install`. If it uses `nsp check` under the covers then Dependency-Check should have the same results, if it does something else, then Dependency-Check doesn't do it.

**Comment by jeremylong** [comment]
I just performed a quick test:

```shell
> npm install jsonwebtoken@0.3.0
npm WARN deprecated jsonwebtoken@0.3.0: Critical vulnerability fix in v5.0.0. See https://auth0.com/blog/2015/03/31/critical-vulnerabilities-in-json-web-token-libraries/
npm WARN deprecated jws@0.2.6: Security update: Versions below 3.0.0 are deprecated.
npm_test@1.0.0 C:\Users\jerem\projects\npm_test
`-- jsonwebtoken@0.3.0
  `-- jws@0.2.6
    +-- base64url@0.0.6
    `-- jwa@0.0.1
```

So `npm install` flags this.  However, if you then run:
```shell
> nsp check
(+) No known vulnerabilities found
```

So currently expected results are that no findings would show up from dependency-check
